### DISCUSSION TITLE
Looking to Access the Aggregator (I'm the BMLT Guy)

### DISCUSSION THREAD

**Discussion by ChrisMarshallNY** [code4recovery/12-step-meeting-list#463] [General]
Hey folks,

I tried contacting you through channels (GSO, etc.), but no response, so I'm giving this venue a stab.

Hi, I'm Chris M. I've been working on Recovery tech since the mid-90s, and was the original architect of [the BMLT](https://bmlt.app) (I hardly have anything to do with it anymore).

I'm working with a 501(c)(3) team, now, on an app that aggregates meetings, and meeting-makers. It is currently NA-only, because of the BMLT, but I really want to integrate AA and Al-Anon.

For the BMLT, I have created [the BMLTiOSLib](https://github.com/bmlt-enabled/BMLTiOSLib), which is a native-Swift SDK. I point it at [TOMATO](https://tomato.bmltenabled.org/main_server/), which is a worldwide BMLT aggregator, and that gives my app (I write native Swift, for Apple devices) the ability to easily find meetings throughout the world.

I'd like to write something similar for the AA system, and integrate it into the app.

Could someone help me out, here?

You can reach me easily. I tend to avoid giving my email out directly, but if you append a "@" to "cmarshall," and then suffix that with "mac," followed by "." and "com," you might have some luck.

Cheers!

**Comment by 97eighty** [comment]
Hi Chris. In the interest of full disclosure, which 501(c)(3) team are you working with?

**Reply by ChrisMarshallNY** [reply]
I've actually already been told that I'm not getting that information.

I have to ask AAWS for permission (which is actually fine, and understandable).

So I think I should just avoid this thread. I'm barking up the wrong tree.

Please forgive the intrusion.
